src/main/java/commands/CommandList.java

```diff
@@ -0,0 +1,36 @@
+package commands;
+
+import processor.DukeProcessor;
+import tasks.Task;
+
+import java.util.List;
+
+/**
+ * Command that prints out the list of tasks as entered by the user, in the order that they were entered.
+ */
+public class CommandList implements Command {
+
+    /**
+     * Prints the list of tasks.
+     *
+     * @param processor The instantiated DukeProcessor object.
+     * @param args      The arguments as entered by the user.
+     */
+    public String execute(DukeProcessor processor, String args) {
+        List<Task> taskList = processor.getTaskList().getTasks();
+        if (taskList.size() == 0) {
+            return "Looks like you don't have any tasks entered! Try entering one with the "
+                    + "commands 'todo', 'deadline' or 'event'!";
+        } else {
```

Comment:
do you really need the else here?
consider eliminating it to make the happy path prominent.